### PR TITLE
CsmSerialNumber.trn - typo and PVL compilant fix

### LINKED ISSUE #5484: CSM Translation File - Not PVL Compliant?
**ISIS version(s) affected**: 8.0.0+

**Description**  

The CSM serial number translation file is not parsable by other PVL libraries. As noted in [this issue](https://github.com/planetarypy/pvl/issues/108) on the planetarypy PVL library, this PVL is not compliant with a PVL specification.

Can this PVL be made compliant and maintain the same functionality? In order to make this work in the PVL library, "Group" inside the `Group = /End_Group` was quoted and the bare `Auto` keyword (??) was removed.

Here is the ISIS PVL file: https://github.com/DOI-USGS/ISIS3/blob/21308692275a243ff58aeefce996cb3c0d8c357b/isis/appdata/serialnumbers/CsmSerialNumber.trn#L4

**How to reproduce**  
This reproduces in an external library, so maybe this is behaving as intended. I hope not, because libraries like plio make use of these translation tables and need a mechanism to convert them into JSON. Right now, the PVL library is that means.

**Possible Solution**  

**Additional context**

### LINKED ISSUE #5485: CSM SerialNumber Translation Defines PlatformId, Labels Use PlatformID
**ISIS version(s) affected**: 8.0.0+

**Description**  

Serial number generation when `csminit`ed use `PlatformId` as the first key. The labels define `PlatformID`. It appears that ISIS is treating these keys in a case-insensitive way. Other applications do not (e.g., plio). Which is the canonical form of the key?

**How to reproduce**  

Translation table: https://github.com/DOI-USGS/ISIS3/blob/21308692275a243ff58aeefce996cb3c0d8c357b/isis/appdata/serialnumbers/CsmSerialNumber.trn#L1-L9

Label snippet:

```
  Group = CsmInfo
    CSMPlatformID       = SELENE
    CSMInstrumentId     = "TERRAIN CAMERA 2"
    ReferenceTime       = 2008-12-08T00:10:22Z
    ModelParameterNames = ("IT Pos. Bias   ", "CT Pos. Bias   ",
                           "Rad Pos. Bias  ", "IT Vel. Bias   ",
                           "CT Vel. Bias   ", "Rad Vel. Bias  ",
                           "Omega Bias     ", "Phi Bias       ",
                           "Kappa Bias     ", "Omega Rate     ",
                           "Phi Rate       ", "Kappa Rate     ",
                           "Omega Accl     ", "Phi Accl       ",
                           "Kappa Accl     ", "Focal Bias     ")
    ModelParameterUnits = (m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m)
    ModelParameterTypes = (REAL, REAL, REAL, REAL, REAL, REAL, REAL, REAL,
                           REAL, REAL, REAL, REAL, REAL, REAL, REAL, REAL)
  End_Group
```
**Possible Solution**  

**Additional context**

### PR DESCRIPTION
**Pull request by amystamile-usgs**
## Related Issue

closes #5485 
closes #5484 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added my user impacting change to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document. 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by amystamile-usgs** [comment]
Looking into `CombineNetworks.FunctionalTestCnetcombineptList` test failure.

File: isis/appdata/serialnumbers/CsmSerialNumber.trn [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Group = Keyword1
-  Auto
-  InputKey       = CSMPlatformId
+  Auto           = 0|1
```
Comment (acpaquette) [inline isis/appdata/serialnumbers/CsmSerialNumber.trn]: This should be Auto = 1 for true. This should also propagate to all `.trn` files in ISIS

**Review by acpaquette — COMMENTED** [review]
Overall looks like a good start. The changes to the `Auto` keyword should likely propagate to all `.trn` files in ISIS

**Comment by amystamile-usgs** [comment]
> Overall looks like a good start. The changes to the `Auto` keyword should likely propagate to all `.trn` files in ISIS

I may push a separate PR for the other translation files.

**Review by acpaquette — APPROVED** [review]

**Pull request merged**